### PR TITLE
compatibility with protoc 2.6.1

### PR DESCRIPTION
**Pull request by pm-nsimic**

### CODE REVIEW

**Comment by pm-nsimic** [comment]
the changes allow the plugin to compile and run with protobufs 2.6.1

File: Makefile [hunk 1]
```diff
@@ -72,13 +72,13 @@ dist/options.proto: options.proto | dist
 dist/protoc-gen-as3: dist/protoc-gen-as3.jar dist/protobuf-java-$(PROTOBUF_VERSION).jar \
 	| dist
 	(echo '#!/bin/sh';\
-	echo 'cd "`dirname "$$0"`" && java -jar protoc-gen-as3.jar') > $@
+	echo 'cd "`dirname "$$0"`" && java -cp "protoc-gen-as3.jar:protobuf-java-$(PROTOBUF_VERSION).jar" com.netease.protocGenAs3.Main') > $@
```
Comment (Atry) [inline Makefile]: Would you mind to change the MANIFEST for classpath instead of here?

**Comment by Atry** [comment]
What is your purpose?

I think the published binary of this plugin built with Protobuf 2.4.1 works with Protobuf 2.6.x

**Comment by pm-nsimic** [comment]
binary maybe, but compiling from source does not work without these changes

**Comment by Atry** [comment]
Recently I added configuration for Travis CI. Could you update your PR to pass the CI build?

**Comment by pm-nsimic** [comment]
The problems are caused by the tests of the 2.6.1 protobufs needing too much memory or cpu:
https://docs.travis-ci.com/user/common-build-problems/#My-build-script-is-killed-without-any-error

I am thinking of pulling the precompiled protoc and jar of version 2.6.1 instead of compiling the entire project. What do you think of that? Any other ideas how i might solve this one?

**Comment by pm-nsimic** [comment]
I disabled the maven tests in the Makefile. It now builds without problems on TravisCI instance

**Pull request closed without merging**